### PR TITLE
fix: not use fs promise, as far as possibile support node 12

### PR DESCRIPTION
**Pull request by zllkjc**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/toolkit/node-bundle-require/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import fs from 'fs/promises';
+import fs from 'fs';
```
Comment (leeight) [inline packages/toolkit/node-bundle-require/src/index.ts:1]: 或者用 `fs-extra` 呢？

Comment (leeight) [reply packages/toolkit/node-bundle-require/src/index.ts:1]: 另外，不是只考虑支持 14.x 么？为什么这里要考虑 12.x 的情况

Comment (zllkjc) [reply packages/toolkit/node-bundle-require/src/index.ts:1]: 这个 pr 我给关了，刚验证了一波，我想岔了

Comment (zllkjc) [reply packages/toolkit/node-bundle-require/src/index.ts:1]: 不对，没想岔。主要是现在很多下游服务还是 v12 的状态，尽可能保持一下。我改成我们 utils 包里导出的 fs-extra

**Review by yimingjfe — APPROVED** [review]

**Pull request merged**